hack/coverage-sanitize.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+
+THIS_DIR="$( dirname "$(realpath "$0")")"
+REPO_DIR="$( realpath ${THIS_DIR}/../ )"
+
+COVER_FILE=${REPO_DIR}/cover.out
+
+if [ ! -f ${COVER_FILE} ]; then
+  echo "${COVER_FILE} not found, nothing to do"
+  exit 1
```

Comment:
is this an error that should fail the build?

Reply:
good question. I would say yes, because it's run mainly from makefile immediately after `go test` which is supposed to create `cover.out`. If it doesn't do that, something went really wrong.
However, I don't have strong opinion about this. If you have good reason not to do that, I'll change it to `0` or handle it in the makefile to not fail in case of non-zero result.